metrics.go

```diff
@@ -549,3 +558,24 @@ func (o *SiteResyncMetrics) Merge(other *SiteResyncMetrics) {
 		*o = *other
 	}
 }
+
+type NetMetrics struct {
+	// Time these metrics were collected
+	CollectedAt time.Time `json:"collected"`
+
+	// net of Interface
+	InterfaceName string `json:"interfaceName"`
+
+	NetStats procfs.NetDevLine `json:"netstats"`
+}
+
+// Merge other into 'o'.
+func (n *NetMetrics) Merge(other *NetMetrics) {
```

Comment:
This must *accumulate*, so new value of `n` represents `n + other`. This way we accumulate for all hosts in the total.